backend/src/main/java/com/htecgroup/skynest/exception/folder/FolderAlreadyDeletedException.java

```diff
@@ -0,0 +1,12 @@
+package com.htecgroup.skynest.exception.folder;
+
+import com.htecgroup.skynest.exception.SkyNestBaseException;
+import org.springframework.http.HttpStatus;
+
+public class FolderAlreadyDeletedException extends SkyNestBaseException {
+  public static final String MESSAGE = "Folder already deleted";
+
+  public FolderAlreadyDeletedException() {
+    super(MESSAGE, HttpStatus.NOT_FOUND);
```

Comment:
Maybe 409 (Conflict) is better here, as FolderNotFound would throw a 404

Reply:
Yes I already change it in my other PR, but I forgot on this one.